### PR TITLE
Adds Optional Links to Content Overrides

### PR DESCRIPTION
**Pull request by felix-hellman**
Screenshot 2023-09-20 at 14 36 25

This is working but I had to do some ugly hacks in frontend to get everything to align properly @oestrogen maybe you can improve on my non-optimal solution?

### CODE REVIEW

File: bankid-idp/env/local/overrides/custom.messages [hunk 1]
```diff
@@ -9,8 +9,15 @@
...
-      "sv": "Detta är en lokal utvecklingsversion av BankID IdP:n",
-      "en": "This is a local development version of the BankID IdP"
+      "sv": "Detta är en lokal utvecklingsversion av BankID IdP:n och du behöver ett BankID för test för att kunna använda den",
+      "en": "This is a local development version of the BankID IdP and you will need a BankID for test in order to use it"
+    }
+  },
+  {
+    "code": "custom.devel.link.text",
+    "text": {
+      "sv": "Skaffa Test BankID",
```
Comment (martin-lindstrom) [inline bankid-idp/env/local/overrides/custom.messages:19]: Ändra till "Skaffa BankID för test". Annars blir det en särskrivning. Vi använder inte heller camel-case på svenska.
/Petig

File: docs/override.md [hunk 2]
```diff
@@ -110,12 +110,36 @@ To show an informational alert box on top of the main content using the custom m
...
+        "text": "my.nocall.text"
+      }
+    ],
+    "position": "ABOVE",
+    "type": "INFO"
+  }
+]
+```
+
+You can also include links in the alert boxes
```
Comment (martin-lindstrom) [inline docs/override.md:126]: end with ':'.

File: docs/override.md [hunk 3]
```diff
@@ -110,12 +110,36 @@ To show an informational alert box on top of the main content using the custom m
 
 ```json
```
Comment (martin-lindstrom) [inline docs/override.md:111]: Should also state that each content element will become a paragraph in the alert box ...

**Review by martin-lindstrom — CHANGES_REQUESTED** [review]

**Comment by oestrogen** [comment]
I fixed the styling and also added back the dot-files for ESLint and Prettier that where lost in the restructuring.

**Review by oestrogen — APPROVED** [review]

**Review by martin-lindstrom — APPROVED** [review]

**Pull request merged**